tests/e2e/tests/critical-path/monitor/monitor.e2e.ts

```diff
@@ -0,0 +1,83 @@
+import { addNewStandaloneDatabase } from '../../../helpers/database';
```

Comment:
Are you sure you don't have import testcafe?

Reply:
I am not sure that understood your question. Could you please rephrase?